books/free-programming-books-bn.md

```diff
@@ -20,6 +20,7 @@
 
 ### C
 
+* [কম্পিউটার প্রোগ্রামিং](http://cpbook.subeen.com/p/blog-page.html) - Tamim Shahriar Subeen
```

Comment:
This appears to be a blog sharing the book's introduction, not the full book

Reply:
It is a full book. If you kindly visit this page http://cpbook.subeen.com/